examples/python/hellostreamingworld/async_greeter_client.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Python AsyncIO implementation of the GRPC hellostreamingworld.MultiGreeter client."""
+
+import asyncio
+import logging
+
+import grpc
+import hellostreamingworld_pb2
+import hellostreamingworld_pb2_grpc
+
+
+async def run() -> None:
+    async with grpc.aio.insecure_channel('localhost:50051') as channel:
+        stub = hellostreamingworld_pb2_grpc.MultiGreeterStub(channel)
+        async for response in stub.sayHello(hellostreamingworld_pb2.HelloRequest(name='you')):
+            print("Greeter client received: " + response.message)
```

Comment:
optional: Can you add one more section with the `stub.read()`?

Reply:
Done. Seems ok for you?

Reply:
Yep. Looks good.